google-cloud-storage/src/test/java/com/google/cloud/storage/ITOpenTelemetryTestbenchTest.java

```diff
@@ -73,12 +73,20 @@ public void setUp() {
                     .addSpanProcessor(SimpleSpanProcessor.create(exporter))
                     .build())
             .build();
-    options = storage.getOptions().toBuilder().setOpenTelemetrySdk(openTelemetrySdk).build();
+    StorageOptions options =
+        storage.getOptions().toBuilder().setOpenTelemetry(openTelemetrySdk).build();
     storage = options.getService();
     String objectString = generator.randomObjectName();
     blobId = BlobId.of(testBucket.getName(), objectString);
   }
 
+  @After
+  public void tearDown() throws Exception {
+    if (storage != null) {
+      storage.close();
```

Comment:
make sure we close the constructed storage instance so we don't leak grpc clients